### PR TITLE
add testcase that check yield malformed yaml errors

### PR DESCRIPTION
**Pull request by charles-chenzz**
add tests that check accumulateResources errors

### CODE REVIEW

File: api/krusty/accumulation_test.go [hunk 1]
```diff
@@ -283,6 +297,14 @@ resources:
 			// know resource is file.
 			errDir: `new root '%s' cannot be absolute`,
 		},
+		{
+			name:     "yaml error",
```
Comment (stormqueen1990) [inline api/krusty/accumulation_test.go:301]:
```suggestion
			name:     "malformed yaml yields an error",
```

**Review by stormqueen1990 — COMMENTED** [review]
This LGTM, just leaving a small suggestion for the test case name.

File: api/krusty/accumulation_test.go (hunk 1 above)
Comment (charles-chenzz) [reply api/krusty/accumulation_test.go]: updated. PTAL

**Review by stormqueen1990 — COMMENTED** [review]
/lgtm

**Pull request merged**